File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.js
function solution(num_list) {
 let lastNum = num_list[num_list.length - 1];
 let prevNum = num_list[num_list.length - 2];
 let result =  lastNum > prevNum ? lastNum - prevNum : lastNum * 2;
 num_list.push(result);
    return num_list;
}

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    
    last_number = num_list[-1]
    compair_number = num_list[-2]
    
    if last_number > compair_number:
        num_list.append(last_number - compair_number)
    else:
        num_list.append(last_number*2)
    
    return num_list